extension.neon

```diff
@@ -22,6 +22,8 @@ parameters:
 		- stubs/ObjectManager.stub
 		- stubs/ObjectManagerDecorator.stub
 		- stubs/ObjectRepository.stub
+		- stubs/ObjectRepository.stub
```

Comment:
This stub no longer exists, probably wrong rebase.

Reply:
Sorry, I think I've fixed